project/src/components/app/app.tsx

```diff
@@ -1,4 +1,13 @@
-import MainPage from '../main-page/main-page';
+import {BrowserRouter, Route, Routes} from 'react-router-dom';
+import {AppRoute, AuthorizationStatus} from '../../const';
+import SignIn from '../../pages/sign-in/sign-in-page';
+import MyList from '../../pages/my-list/my-list-page';
```

Comment:
Папки, в которых лежат компоненты должны называться также, как и сам компонент.
```suggestion
import SignIn from '../../pages/sign-in-page/sign-in-page';
import MyList from '../../pages/my-list-page/my-list-page';
```